src/test/java/fi/otavanopisto/kuntaapi/test/ManagementMocker.java

```diff
@@ -150,24 +163,39 @@ private Announcement readAnnouncementFromJSONFile(String file) {
     return readJSONFile(file, Announcement.class);
   }
   
+  /**
+   * Reads JSON file as fragment object
+   * 
+   * @param file path to JSON file
+   * @return read object
+   */    
+  private Fragment readFragmentFromJSONFile(String file) {
+    return readJSONFile(file, Fragment.class);
+  }
+
   @Override
   public void startMock() {
-    Map<String, String> pageQuery = new HashMap<>();
-    pageQuery.put("per_page", "100");
+    Map<String, String> pageQuery100 = new HashMap<>();
+    pageQuery100.put("per_page", "100");
+    Map<String, String> pageQuery100_1 = new HashMap<>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00117)